Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM ruby:2.3.3
```

Comment:
If you want, we can target ruby 2.4.0. With docker, the mac issue would be irrelevant. Currently our web container is based on a debian image.